src/components/DataGrid/DataGrid.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import {
+  DataGrid as MuiDataGrid,
+  DataGridProps,
+  GridColDef,
+  GridRowsProp,
+} from '@mui/x-data-grid';
+
+import theme from '../../theme';
+import { makeStyles } from '../../theme';
+import Box from '../Box';
+
+export interface Props extends DataGridProps {
+  columns: GridColDef[];
+  rows: GridRowsProp;
+  pageSize?: number;
+  height: string;
```

Comment:
the former two are more straightforward but could you add comments above these two props for what they do? (`pageSize` and `height`)

Reply:
and also using the `@default` syntax we use in a few places for the `pageSize` prop